packages/extension-browser/src/devtools/views/pages/error.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { ErrorData } from '../../../shared/types';
+
+import { getMessage } from '../../utils/i18n';
+
+import Page from '../page';
+
+import * as styles from './error.css';
+
+const openIssueUrl = 'https://github.com/webhintio/hint/issues/new?labels=type%3Abug&amp;template=1-bug-report.md&amp;title=%5BBug%5D+Bug+description';
```

Comment:
I think we should create a template for the browser extension. What do you think?

Reply:
I like that - I'll draft one up as part of this PR.

Reply:
Done!